MediaBrowser.MediaEncoding/Attachments/AttachmentExtractor.cs

```diff
@@ -178,7 +178,7 @@ await ExtractAttachment(inputFile, protocol, mediaAttachment.Index, outputPath,
 
                 process.Start();
 
-                var ranToCompletion = await process.WaitForExitAsync(cancellationToken).ConfigureAwait(false);
+                var ranToCompletion = await ProcessExtensions.WaitForExitAsync(process, cancellationToken).ConfigureAwait(false);
```

Comment:
`Process` now has a `WaitForExitAsync` extension, so call the internal extension
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.waitforexitasync?view=net-5.0